terraform/ecs_asg/variables.tf

```diff
@@ -51,3 +51,11 @@ variable "admin_cidr_ingress" {
   default     = "0.0.0.0/0"
   description = "CIDR for SSH access to EC2 instances"
 }
+
+variable "sns_topic_arn" {
+  description = "ARN of the topic where to push notifications when an EC2 instance is set to terminating state"
+}
+
+variable "publish_to_sns_policy" {
+  description = "Policy document to publish to the sns topic"
```

Comment:
Is this something that's going to be published _to_ the SNS document, or something that allows us to publish to the SNS topic? I’m not quite clear.